federation-cdn-mock/webpack.config.js

```diff
@@ -53,18 +53,47 @@ const TestSDKPLugin = new DynamicRemotePlugin({
   },
 });
 
+const FullManifest = new DynamicRemotePlugin({
+  extensions: [],
+  sharedModules,
+  pluginManifestFilename: 'full-manifest.json',
+  entryScriptFilename: 'full-manifest.js',
+  moduleFederationSettings: {
+    // Use non native webpack plugins
+    pluginOverride: {
+      ModuleFederationPlugin,
+      ContainerPlugin,
+    },
+  },
+  pluginMetadata: {
+    name: 'full-manifest',
+    version: '1.0.0',
+    exposedModules: {
```

Comment:
You probably want to include just the one JS module you are using in the tests. Don't need to build everything twice.